java.io.IOException: Cannot run program "which": CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at scala.sys.process.ProcessBuilderImpl$Simple.run(ProcessBuilderImpl.scala:68)
	at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.lines(ProcessBuilderImpl.scala:140)
	at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.lines_$bang(ProcessBuilderImpl.scala:109)
	at scala.scalanative.build.Discover$.discover(Discover.scala:156)
	at scala.scalanative.build.Discover$.clangpp(Discover.scala:31)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6$$anonfun$apply$2.apply(ScalaNativePluginInternal.scala:57)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6$$anonfun$apply$2.apply(ScalaNativePluginInternal.scala:57)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$.scala$scalanative$sbtplugin$ScalaNativePluginInternal$$interceptBuildException(ScalaNativePluginInternal.scala:204)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6.apply(ScalaNativePluginInternal.scala:57)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6.apply(ScalaNativePluginInternal.scala:57)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
	at java.lang.ProcessImpl.start(ProcessImpl.java:137)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	at scala.sys.process.ProcessBuilderImpl$Simple.run(ProcessBuilderImpl.scala:68)
	at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.lines(ProcessBuilderImpl.scala:140)
	at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.lines_$bang(ProcessBuilderImpl.scala:109)
	at scala.scalanative.build.Discover$.discover(Discover.scala:156)
	at scala.scalanative.build.Discover$.clangpp(Discover.scala:31)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6$$anonfun$apply$2.apply(ScalaNativePluginInternal.scala:57)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6$$anonfun$apply$2.apply(ScalaNativePluginInternal.scala:57)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$.scala$scalanative$sbtplugin$ScalaNativePluginInternal$$interceptBuildException(ScalaNativePluginInternal.scala:204)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6.apply(ScalaNativePluginInternal.scala:57)
	at scala.scalanative.sbtplugin.ScalaNativePluginInternal$$anonfun$scalaNativeBaseSettings$6.apply(ScalaNativePluginInternal.scala:57)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[0m[[31merror[0m] [0m(myProject0/*:[31mnativeClangPP[0m) java.io.IOException: Cannot run program "which": CreateProcess error=2, The system cannot find the file specified[0m
